tests/Yaapii.Atoms.Tests/Map/MapEnvelopeTest.cs

```diff
@@ -47,6 +48,24 @@ public void TryGetValueWithMissingKey()
             Assert.False(map.TryGetValue(0, out outValue));
         }
 
+        [Fact]
+        public void GetValueWithExistingKey()
```

Comment:
We want to write method names as a „fact“ about the class - so here. It should be „Gets“